.github/workflows/prod-build.yml

```diff
@@ -0,0 +1,71 @@
+name: 'Production Release'
+
+on:
+  workflow_dispatch:
+    inputs:
+      package:
+        required: true
+        type: choice
+        description: Which package should be published?
+        options:
+          - vue
+          - angular
+          - react
+      version:
+        required: true
+        type: choice
+        description: Which version should be published?
+        options:
+          - patch
+          - minor
+          - major
+          - prepatch
+          - preminor
+          - premajor
+          - prerelease
+      tag:
+        required: true
+        type: choice
+        description: Which npm tag should this be published to?
+        options:
+          - latest
+          - next
+      preid:
+        type: choice
+        description: Which prerelease identifier should be used? This is only needed when version is "prepatch", "preminor", "premajor", or "prerelease".
+        options:
+          - ''
+          - alpha
+          - beta
+          - rc
+          - next
+
+jobs:
+  dev-build:
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.dev-build.outputs.version }}
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 16
+      - name: Install Dependencies
+        run: npm ci --no-package-lock && lerna bootstrap --ignore-scripts -- --legacy-peer-deps
```

Comment:
May not be necessary, depending on if we need this step - if we do need to run this, does it make sense to use the `--no-package-lock` flag in conjunction with `npm ci`? I _think_ it would be a no-op flag, but I've also never tried to use it in this way and could be misunderstanding what it does

Reply:
We need `npm ci` to have access to lerna. I've also never used `--no-package-lock` - this was basically a fork from framework's own workflow so I thought there might be a lerna-y reason for why it's passed in.

Reply:
@liamdebeasi Do you think we need `--no-package-lock` here?

Reply:
No, I think `--no-package-lock` in Ionic Framework is left over from when we were migrating from our custom release scripts to Lerna.

Reply:
👍 TY!